app/javascript/pages/map/data-analysis-menu/component.jsx

```diff
@@ -67,6 +69,8 @@ class DataAnalysisMenu extends PureComponent {
               analysis.showResults && <PolygonAnalysis />}
           </div>
         )}
+        <CountryDataProvider location={analysis.location} />
```

Comment:
We are likely going to need country data outside of the analysis on the map in many cases. Think embed forms etc. We could leave it here, but maybe it is clearer at the page level?